src/utils/request/fetch.ts

```diff
@@ -170,8 +191,8 @@ export default function fetchRequest(
     options.url,
     fetchOpts
   ).then((response : Response) : Promise<IFetchedStreamComplete> => {
-    if (!isNullOrUndefined(timeout)) {
-      clearTimeout(timeout);
+    if (!isNullOrUndefined(connectionTimeoutId)) {
```

Comment:
I know that's what was here before, but no need to check for `isNullOrUndefined` if it can only be `undefined` or a number

Reply:
true :)